src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,58 @@
+import "./Footer.css";
+const Footer = () => {
+  return (
+    <footer className="footer">
+      <div className="footer-container">
+        <div className="footer-column">
+          <h4 className="footer-col-heading">Our Company</h4>
+          <ul className="footer-list">
+            <li className="footer-list-item">About Us</li>
+            <li className="footer-list-item">Career</li>
+            <li className="footer-list-item">Our Services</li>
+            <li className="footer-list-item">Faq</li>
+          </ul>
+        </div>
+        <div className="footer-column">
+          <h4 className="footer-column-heading">Categories</h4>
+          <ul className="footer-list">
+            <li className="footer-list-item">Chairs & Sofas</li>
+            <li className="footer-list-item">Lamp & Lightings</li>
+            <li className="footer-list-item">Storage & Shelves</li>
+            <li className="footer-list-item">Living Room</li>
+          </ul>
+        </div>
+        <div className="footer-column">
+          <h4 className="footer-col-heading">Useful Links</h4>
```

Comment:
suggestion - you can store this data in array of objects and map over it. It will look clean.